tests/Binance/SignerTests.cpp

```diff
@@ -202,4 +202,122 @@ TEST(BinanceSigner, BuildSend2) {
     );
 }
 
+    TEST(BinanceSigner, BuildHTLT) {
+        const auto derivationPath = TW::derivationPath(TWCoinTypeBinance);
+
+        const auto fromWallet = HDWallet("patrol tired convince excite giant list tissue spring venture between rubber child kite hurry bulk tired raccoon beyond wrist naive message buzz ice slice", "");
+        const auto fromPrivateKey = fromWallet.getKey(derivationPath);
+        const auto fromPublicKey = PublicKey(fromPrivateKey.getPublicKey(TWPublicKeyTypeSECP256k1));
+        auto fromAddr = Cosmos::Address(HRP_BINANCE, fromPublicKey).keyHash;
+
+        const auto toWallet = HDWallet( "recycle poet history join say present until laptop normal burger inject entire end beach talent occur depth ask wage purpose tornado hold easily kick", "");
+        const auto toPrivateKey = toWallet.getKey(derivationPath);
+        const auto toPublicKey = PublicKey(toPrivateKey.getPublicKey(TWPublicKeyTypeSECP256k1));
+        auto toAddr = Cosmos::Address(HRP_BINANCE, toPublicKey).keyHash;
+
```

Comment:
can you use one private key for all examples? no need to convert from mnemonic => private key